generators/app/templates/package.json

```diff
@@ -39,50 +39,53 @@
   },
   "homepage": "https://github.com/ship-components/generator-ship-components-boilerplate#readme",
   "dependencies": {
-    "babel-runtime": "^6.23.0",
-    "classnames": "^2.2.5",
-    "immutable": "^3.8.1",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+    "classnames": "^2.2.6",
+    "immutable": "^3.8.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
   },
   "devDependencies": {
-    "autoprefixer": "^7.1.1",
-    "@babel/core": "^6.25.0",
-    "@babel/eslint": "^7.2.3",
-    "@babel/loader": "^7.1.1",
-    "@babel/plugin-transform-runtime": "^6.23.0",
-    "@babel/preset-env": "^1.6.0",
-    "@babel/preset-react": "^6.24.1",
-    "coveralls": "^2.13.1",
-    "css-loader": "^0.28.4",
-    "enzyme": "^2.9.1",
-    "enzyme-to-json": "^1.5.1",
-    "eslint": "^4.1.1",
-    "eslint-config-ship-components": "^1.4.2",
-    "eslint-loader": "^1.8.0",
-    "eslint-plugin-react": "^7.3.0",
-    "extract-text-webpack-plugin": "^2.1.2",
-    "file-loader": "^0.11.2",
-    "grunt": "^1.0.1",
-    "grunt-contrib-clean": "^1.1.0",
-    "grunt-coveralls": "^1.0.1",
-    "grunt-eslint": "^20.0.0",
-    "grunt-webpack": "^3.0.0",
-    "jest": "^20.0.4",
-    "jest-css-modules": "^1.1.0",
-    "package-banner": "^0.1.1",
-    "postcss-calc": "^5.3.1",
-    "postcss-color-function": "^4.0.0",
-    "postcss-color-hex-alpha": "^3.0.0",
-    "postcss-loader": "^2.0.6",
-    "postcss-nested": "^2.0.2",
-    "postcss-simple-vars": "^4.0.0",
-    "prop-types": "^15.5.10",
-    "react-addons-test-utils": "^15.6.0",
-    "react-test-renderer": "^15.6.1",
-    "style-loader": "^0.18.2",
-    "webpack": "^3.2.0",
-    "webpack-dev-server": "^2.5.0"
+    "@babel/core": "^7.4.5",
+    "@babel/plugin-transform-runtime": "^7.4.4",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-react": "^7.0.0",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.6",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-react": "^6.24.1",
+    "babel-runtime": "^6.26.0",
```

Comment:
This one should actually be in the dependencies see https://babeljs.io/docs/en/babel-runtime#usage